.github/workflows/pr.yml

```diff
@@ -73,14 +75,62 @@ jobs:
+        run: |
+          SERVICE_BOX_KUBERNETES_NAMESPACE=${{ env.SERVICE_BOX_KUBERNETES_NAMESPACE }}
+          POD_NAME=$(kubectl get pod -l app.kubernetes.io/name=servicebox-app -n $SERVICE_BOX_KUBERNETES_NAMESPACE -o name)
+          KIND_URL=https://kubernetes.default.svc
+          # To connect to Kind from outside a pod:
+          # KIND_URL=$(kubectl config view -o jsonpath='{"Cluster name\tServer\n"}{range .clusters[*]}{.name}{"\t"}{.cluster.server}{"\n"}{end}' | grep kind-kind | sed "s/kind-kind//" | xargs)
+          kind get kubeconfig > /tmp/local-kind-kubeconfig
+          kubectl cp /tmp/local-kind-kubeconfig sb/${POD_NAME:4}:/tmp/local-kind-kubeconfig -c servicebox-app
+          RESULT=$(kubectl exec -i $POD_NAME --container servicebox-app -n $SERVICE_BOX_KUBERNETES_NAMESPACE -- sh -c "curl -X POST -H 'Content-Type: multipart/form-data' -H 'HX-Request: true' -F name=local-kind -F environment=DEV -F url=$KIND_URL -F kubeConfig=@/tmp/local-kind-kubeconfig -s -i localhost:8080/clusters")
+          if [ "$RESULT" = *"500 Internal Server Error"* ]
+          then
+            echo "Cluster failed to be saved in Service Box: $RESULT"
+            kubectl describe $POD_NAME -n $SERVICE_BOX_KUBERNETES_NAMESPACE
+            kubectl logs $POD_NAME -n $SERVICE_BOX_KUBERNETES_NAMESPACE
+            exit 1
+          fi
+          echo "Cluster installed in Service Box: $RESULT"
+      - name: Create Postgresql Service in Service Box
+        run: |
+          SERVICE_BOX_KUBERNETES_NAMESPACE=${{ env.SERVICE_BOX_KUBERNETES_NAMESPACE }}
+          POD_NAME=$(kubectl get pod -l app.kubernetes.io/name=servicebox-app -n $SERVICE_BOX_KUBERNETES_NAMESPACE -o name)
+
+          # First, we install Postgresql via Helm.
+          helm repo add bitnami https://charts.bitnami.com/bitnami
+          helm install postgresql bitnami/postgresql --version 11.9.1 --set auth.username=superman --set auth.password=superman
+          ## the service endpoint should be: "postgresql:5432"
+
+          # Next, we register the Postgresql service in Service Box
+          BODY="{\"name\": \"postgresql\", \"version\": \"11\", \"endpoint\": \"postgresql:5432\"}"
+          RESULT=$(kubectl exec -i $POD_NAME --container servicebox-app -n $SERVICE_BOX_KUBERNETES_NAMESPACE -- sh -c "curl -X POST -H 'Content-Type: application/json' --data '$BODY' -s -i localhost:8080/services")
+          if [ "$RESULT" = *"500 Internal Server Error"* ]
+          then
+            echo "Service failed to be saved in Service Box: $RESULT"
+            kubectl describe $POD_NAME -n $SERVICE_BOX_KUBERNETES_NAMESPACE
+            kubectl logs $POD_NAME -n $SERVICE_BOX_KUBERNETES_NAMESPACE
+            exit 1
+          fi
+          echo "Service installed in Service Box: $RESULT"
+
+          SERVICE=$(kubectl exec -i $POD_NAME --container servicebox-app -n $SERVICE_BOX_KUBERNETES_NAMESPACE -- sh -c "curl -H 'Accept: application/json' -s localhost:8080/services/name/postgresql")
```

Comment:
The existing github job assumes that post creation of the service `postgresql`, job will run before you will execute such a test 
```
          SERVICE=$(kubectl exec -i $POD_NAME --container servicebox-app -n $SERVICE_BOX_KUBERNETES_NAMESPACE -- sh -c "curl -H 'Accept: application/json' -s localhost:8080/services/name/postgresql")
          if [ $(echo "$SERVICE" | jq -r '.deployed') = "true" ]
```
What about if the job has not yet been executed and then we should wait or retry x times before to be sure that we will got the status deployed = true ?